experimental/add_to_app/android_fullscreen/app/build.gradle

```diff
@@ -5,35 +5,37 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileOptions {
-        sourceCompatibility 1.8
-        targetCompatibility 1.8
-    }
     compileSdkVersion 28
+    buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "com.example.simpleandroidapp"
+        applicationId "dev.flutter.example.androidfullscreen"
         minSdkVersion 19
         targetSdkVersion 28
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
 }
 
 dependencies {
-    implementation 'android.arch.lifecycle:common-java8:1.1.1'
+    implementation project(':flutter')
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
-    implementation project(':flutter')
+    implementation 'android.arch.lifecycle:common-java8:1.1.1'
```

Comment:
sorry, @blasten I was commenting on the wrong line. I actually meant this line. This is no longer needed right?

Reply:
I may have been building against `dev` when I added this line. It's not needed on `master`, so I've removed it.